Description: Adds a OSLO_PACKAGE_VERSION in the setup.py
 Other Openstack packages have a OSLO_PACKAGE_VERSION so that it doesn't
 rely on Git or a tarball to know which version is in use. It seems that
 python-swiftclient is lagging behind, so I'm adding the feature.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2013-04-03

--- python-swiftclient-1.3.0.orig/swiftclient/openstack/common/setup.py
+++ python-swiftclient-1.3.0/swiftclient/openstack/common/setup.py
@@ -302,6 +302,9 @@ def get_version(package_name):
     to make a source tarball from a fork of our repo with additional tags in it
     that they understand and desire the results of doing that.
     """
+    version = os.environ.get("OSLO_PACKAGE_VERSION", None)
+    if version:
+        return version
     version = get_version_from_pkg_info(package_name)
     if version:
         return version
